client/python/kserve-api/samples/README.md

```diff
@@ -0,0 +1,473 @@
+# KServe API usage samples{#ovms_docs_kserve_samples}
+
+OpenVINO Model Server 2022.2 release introduced support for [KServe API](https://github.com/kserve/kserve/tree/master/docs/predict-api/v2).
+
+This guide shows how to interact with KServe API endpoints on both gRPC and HTTP interfaces. It covers following topics:
+- <a href="#grpc-api">GRPC API Examples </a>
+  - <a href="#grpc-server-live">grpc_server_live.py</a>
+  - <a href="#grpc-server-ready">grpc_server_ready.py</a>
+  - <a href="#grpc-server-metadata">grpc_server_metadata.py</a>
+  - <a href="#grpc-model-ready">grpc_model_ready.py</a>
+  - <a href="#grpc-model-metadata">grpc_model_metadata.py</a>
+  - <a href="#grpc-model-infer">grpc_infer_resnet.py</a>
+- <a href="#http-api">HTTP API Example</a>
+  - <a href="#http-server-live">http_server_live.py</a>
+  - <a href="#http-server-ready">http_server_ready.py</a>
+  - <a href="#http-server-metadata">http_server_metadata.py</a>
+  - <a href="#http-model-ready">http_model_ready.py</a>
+  - <a href="#http-model-metadata">http_model_metadata.py</a>
+  - <a href="#http-model-infer">http_infer_resnet.py</a>
+
+> **Note:** Some of the samples will use [ResNet50](https://github.com/openvinotoolkit/open_model_zoo/blob/2022.1.0/models/intel/resnet50-binary-0001/README.md).
+
+## Before you run the samples
+
+### Clone OpenVINO&trade; Model Server GitHub repository and enter model_server directory.
+```Bash
+git clone https://github.com/openvinotoolkit/model_server.git
+cd model_server
+```
+### Download the Pretrained Model
+Download the model files and store them in the `models` directory
+```Bash
+mkdir -p models/resnet/1
+curl https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.bin https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.xml -o models/resnet/1/resnet50-binary-0001.bin -o models/resnet/1/resnet50-binary-0001.xml
+```
+
+### Pull the Latest Model Server Image
+Pull the latest version of OpenVINO&trade; Model Server from Docker Hub :
+```Bash
+docker pull openvino/model_server:latest
+```
+
+### Start the Model Server Container with Downloaded Model and Dynamic Batch Size
+Start the server container with the image pulled in the previous step and mount the `models` directory :
+```Bash
+docker run --rm -d -v $(pwd)/models:/models -p 9000:9000 -p 5000:5000 openvino/model_server:latest --model_name resnet --model_path /models/resnet --batch_size auto --port 9000 --rest_port 5000
```

Comment:
we expect that auto batch_size will be deprecated soon so it might be better to stop advertising it. For the use case with binary input, it might be useful to add --layout parameter to set it to NHWC.